### DISCUSSION TITLE
Webdav server password

### DISCUSSION THREAD

**Discussion by enboig** [rustic-rs/rustic#1271] [Ideas]
I think there should be a way to set a password for webdav server; now any user in your intranet have access to your files when you spawn a webdav server. It could be repo password or some password set in config file.

**Comment by aawsome** [comment]
I think this is related to adding TLS support for webdav. If we implement a simple user/password authentication, this doesn't help much if on your intranet others can see the trafic and just read the credentials...

But anyway, webdav is done by the `warp` crate at the moment - if anyone wants to add improvements, they will be very local at the place where `warp` is called..

**Comment by simonsan** [comment]
There was an issue opened about this, where I transferred the information from here: https://github.com/rustic-rs/rustic/issues/1298

So I'll close this discussion, to keep it centralized.
